automatic/dropbox-beta/update.ps1

```diff
@@ -0,0 +1,36 @@
+
+import-module au
+ . "$PSScriptRoot\..\dropbox\update_helper.ps1"
+
+function global:au_BeforeUpdate {
+  $Latest.ChecksumType32 = 'sha256'
+  $Latest.Checksum32 = Get-RemoteChecksum -Algorithm $Latest.ChecksumType32 -Url $Latest.URL32
+  # Copy the original file from the dropbox folder
+  if (!(Test-Path "$PSScriptRoot\tools" -PathType Container)) { New-Item -ItemType Directory "$PSScriptRoot\tools" }
+  Copy-Item "$PSScriptRoot\..\dropbox\tools" "$PSScriptRoot" -Force -Recurse
+  Copy-Item "$PSScriptRoot\..\dropbox\Readme.md" "$PSScriptRoot" -Force -Recurse
+}
+
+function global:au_SearchReplace {
```

Comment:
Since this function is the exact same as the one in the original dropbox package, you can remove the `global:au_SearchReplace` function here.
Please do that now.